check_opnsense.py

```diff
@@ -237,11 +239,37 @@ def check_updates(self) -> None:
         self.perfdata.append("remove_packages={}".format(remove_packages))
         self.perfdata.append("available_updates={}".format(available_updates))
 
+    def check_ipsec(self) -> None:
+        """Check IPsec Status."""
+        url = self.get_url("ipsec/sessions/search_phase1")
+        data = self.request(url)
+
+        if data["total"] > 0:
+            for row in data["rows"]:
+                if not row["connected"]:
+                    self.check_result = CheckState.WARNING
+                    if self.check_message_nok == "":
+                        self.check_message_nok += "'{}' ".format(row["phase1desc"])
+                    else:
+                        self.check_message_nok += ", '{}' ".format(row["phase1desc"])
+                if row["connected"]:
+                    self.check_message_ok += "'{}', ".format(row["phase1desc"])
```

Comment:
```suggestion
                    tunnels_connected.append(row["phase1desc"])
```

Reply:
Try not to build the check message during parsing and just collect the data, so it can be done afterwards.